snippets/cart-drawer.liquid

```diff
@@ -5,7 +5,10 @@
   {% render 'cart-drawer' %}
 {% endcomment %}
 
+{{ 'quantity-popover.css' | asset_url | stylesheet_tag }}
+
 <script src="{{ 'cart.js' | asset_url }}" defer="defer"></script>
+<script src="{{ 'quantity-popover.js' | asset_url }}" defer="defer"></script>
```

Comment:
I was wondering if we could import this conditionally only when there are qty rules or vol pricing but they are set up on the product/variant level and not line item 🤔